lib/rspec/core/configuration.rb

```diff
@@ -908,6 +913,11 @@ def formatter_loader
       end
 
       # @private
+      def output_wrapper
```

Comment:
Does this method need to be public?  While you've marked it private with a YARD annotation it's still public and it would be nice to make it private if possible..